src/python/pants_release/common.py

```diff
@@ -24,12 +25,15 @@ def die(message: str) -> NoReturn:
 
 
 def green(message: str) -> None:
-    print(f"{_COLOR_GREEN}{message}{_COLOR_RESET}")
```

Comment:
I find it just a smiiiiidge funny we didn't inner-source our own ansi color implementation, yet this file... :joy:

Reply:
Fair!